typescript/radix-sdk/src/clients/protocol.ts

```diff
@@ -45,6 +51,42 @@ export class RadixProtocolProvider implements ProtocolProvider {
     throw Error('Not implemented');
   }
 
+  createIsmArtifactManager(
+    chainMetadata: ChainMetadataForAltVM,
+  ): IRawIsmArtifactManager {
+    assert(chainMetadata.gatewayUrls, 'gateway urls undefined');
+
+    const networkId = parseInt(chainMetadata.chainId.toString());
+    const gatewayUrl = chainMetadata.gatewayUrls[0]?.http;
+
+    assert(gatewayUrl, 'gateway url undefined');
+
+    // Get package address from metadata or throw error
+    const packageAddress = (chainMetadata as any).packageAddress;
+    assert(
+      packageAddress,
+      `Expected package address to be defined for radix network with id ${networkId}`,
+    );
```

Comment:
I think that we should export this and use it here because mainnet and testnet metadata in the registry don't have the `packageAddress`

https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/ae8ef43896b1ebe335dcb9c58b1563d60b828e7a/typescript/radix-sdk/src/clients/provider.ts#L57-L71

Reply:
I don't understand. Do we have `packageAddress` in the registry? If not, should we add it?